integration/spark-datasource/src/test/scala/org/apache/spark/sql/carbondata/datasource/TestCreateTableUsingSparkCarbonFileFormat.scala

```diff
@@ -320,6 +324,66 @@ class TestCreateTableUsingSparkCarbonFileFormat extends FunSuite with BeforeAndA
     assert(new File(filePath).exists())
     cleanTestData()
   }
+  test("Read data having multi blocklet ") {
+    buildTestDataMuliBlockLet(700000)
+    assert(new File(writerPath).exists())
+    spark.sql("DROP TABLE IF EXISTS sdkOutputTable")
+
+    if (SparkUtil.isSparkVersionEqualTo("2.1")) {
+      //data source file format
+      spark.sql(s"""CREATE TABLE sdkOutputTable USING carbon OPTIONS (PATH '$writerPath') """)
+    } else if (SparkUtil.isSparkVersionXandAbove("2.2")) {
+      //data source file format
+      spark.sql(
+        s"""CREATE TABLE sdkOutputTable USING carbon LOCATION
+           |'$writerPath' """.stripMargin)
+    } else{
+      // TO DO
```

Comment:
Build will fail for spark 2.3. just keep 2.1 in if check and remaining in else case